### PR TITLE
stream: set connection monitor delay.

### PR DESCRIPTION
**Pull request by dnldd**
# PR Description

- this fixes a bug where the connection monitor delay config value does not get set to the websocket on intialization.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by gloriousCode — APPROVED** [review]
😅 Lovely fix. Thanks for that

**Comment by dnldd** [comment]
Noted, will work that in this weekend.

File: config/config_types.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ const (
 	defaultWebsocketResponseMaxLimit     = time.Second * 7
 	defaultWebsocketOrderbookBufferLimit = 5
 	defaultWebsocketTrafficTimeout       = time.Second * 30
+	defaultConnectionMinitorDelay        = time.Second * 2
```
Comment (shazbert) [inline config/config_types.go:33]: Minitor -> Monitor

File: config/config.go [hunk 2]
```diff
@@ -1821,5 +1821,10 @@ func (c *Exchange) Validate() error {
 	if c == nil {
 		return errExchangeConfigIsNil
 	}
+
+	if c.ConnectionMonitorDelay == 0 || c.ConnectionMonitorDelay < 0 {
```
Comment (shazbert) [inline config/config.go:1825]:
```go
if c.ConnectionMonitorDelay <= 0 {
```

**Review by shazbert — CHANGES_REQUESTED** [review]
Thank you! 🚀 Tested and it works and everything has been set. Just two minor nits.

**Review by gloriousCode — CHANGES_REQUESTED** [review]
Thank you for expanding your PR! One potential suggestion from me

File: config/config_types.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (shazbert) [reply config/config_types.go:33]: Probably need to include websocket: `config.DefaultWebsocketConnectionMonitorDelay`

**Review by gloriousCode — APPROVED** [review]
tACK! Thanks for making those changes

**Review by shazbert — APPROVED** [review]
Changes ACK. thanks 🖖

**Review by thrasher- — APPROVED** [review]
Thank you for your first contribution/fix @dnldd !

**Pull request merged**